src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/MemoryMapLightUp.cs

```diff
@@ -217,6 +217,10 @@ internal static IDisposable CreateViewAccessor(object memoryMap, long start, int
                 s_lazyIsAvailable = false;
                 return null;
             }
+            catch (TargetInvocationException ex) when (ex.InnerException is UnauthorizedAccessException)
+            {
+                throw new IOException(ex.InnerException.Message, ex.InnerException);
+            }
```

Comment:
Does this exception changing and no tests changing indicate that we're missing tests?